src/components/cards/AcceptTXRequestDetailsCard.tsx

```diff
@@ -3,10 +3,9 @@ import React from "react";
 import { Text, View, StyleSheet } from "react-native";
 import { TextInputMask } from "react-native-masked-text";
 
-import { FONTS } from "../../font/fonts/fonts";
+import { FONTS } from "../../styles/fonts/fonts";
```

Comment:
What's the difference between the two?

Reply:
We are using the one in green based on the folder structure